examples/evaluate_different_templates_num_demos.py

```diff
@@ -0,0 +1,30 @@
+import evaluate
+from datasets import load_dataset
+from unitxt.text_utils import print_dict
+
+# Use the HF load_dataset API, to load the wnli entailment dataset using the standard template in the catalog for relation task with 5-shot in-context learning.
+# We set loader_limit to 200 to limit reduce download time.
+# Sample for each example a template from [templates.classification.multi_class.relation.default,templates.key_val]
+# Sample for each example a number of demonstrations from [0,5]
+dataset = load_dataset(
```

Comment:
I think most examples should use the unitxt load_dataset (not the HF api) because it's limited in ability to expand.

Reply:
considering the time frame I suggest to improve the example in a different PR

Reply:
It takes a few minutes.  I think it's worthwhile to do it now and not carry it over.

Reply:
done